v2x_ros_driver/src/v2x_radio_client.cpp

```diff
@@ -168,29 +169,138 @@ void V2XRadioClient::process(const std::shared_ptr<const std::vector<uint8_t>>&
         else {
             // TODO lengths greater than 16383 (0x3FFF) are encoded by splitting up the message into discrete chunks, each with its own length
             // marker. It doesn't look like we'll be receiving anything that long
-            std::cerr << "V2XRadioClient::process() : received a message with length field longer than 16383." << std::endl;
+            RCLCPP_WARN_STREAM(logger_, "V2XRadioClient::process() : received a message with length field longer than 16383.");
```

Comment:
I know the log text is unchanged, but can we add that the message is being discarded?

Reply:
Updated